spyder/plugins/profiler/widgets/main_widget.py

```diff
@@ -173,13 +174,22 @@ def __init__(self, name=None, plugin=None, parent=None):
         self.filecombo = PythonModulesComboBox(
             self, id_=ProfilerWidgetMainToolbarItems.FileCombo)
         self.datatree = ProfilerDataTree(self)
+        self.pane_empty = PaneEmptyWidget(
+            self,
+            "code-profiler",
+            _("No code profiled yet"),
```

Comment:
In the existing Spyder 5 UI, as far as I can tell this cannot be blank unless the user has not profiled any code yet in the session. However, with the UI changes planned or executed for Spyder 6, i.e. those in Quentin's PR, this will by default track the current file instead of just staying on whatever was last profiled (like how the Code Analysis pane works), so we'd presumably want to show this message whenever the user clicked to a file that had been profiled yet. Therefore, **if/when that's the case**, either in this PR or Quentin's, this needs to be changed to (matching the Code Analysis pane):

```suggestion
            _("Code not profiled yet"),
```